AirCastingTests/API/ShareSessionFileTest.swift

```diff
@@ -0,0 +1,27 @@
+// Created by Lunar on 27/07/2022.
+//
+
+import XCTest
+import Resolver
+@testable import AirCasting
+
+class ShareSessionFileTest: APIServiceTestCase {
+    lazy var sut = ShareSessionApi()
+
+    func test_createsCorrectRequest() throws {
+        var receivedRequest: URLRequest?
+        client.requestTaskStub = { request, completion in
+            receivedRequest = request
+            completion(.failure(URLError(.timedOut)), request)
+        }
+        
+        let email = "test@test.com"
+        let uuid = UUID().uuidString
+        sut.sendSession(email: email, uuid: uuid, completion: { _ in })
+
+        let request = try XCTUnwrap(receivedRequest)
+        XCTAssertEqual(request.url, URL(string: "http://aircasting.org/api/sessions/export_by_uuid.json?email=\(email)&uuid=\(uuid)")!)
+        XCTAssertEqual(request.httpMethod, "GET")
```

Comment:
this test is actually testing 2 different things:
URL creation
HTTP method sent

It's very minor here, but it could be split into 2.

The reason is that if we only looked at the tests report we'd only see something like:
```
[PASSED] SomeSuit.test_that_succeeded
[PASSED] SomeSuit.other_test_that_succeeded
[FAILED] ShareSessionFileTest.createCorrectRequset
```
So we wouldn't know why. If we split this into 2:
```
[PASSED] ShareSessionFileTest.whenSendsRequest_hasCorrectURL
[FAILED] ShareSessionFileTest.whenSendsRequest_usesGETMethod
```
Which tells us exactly to the point what is malfunctioning